### ISSUE TITLE
What can I expect if apply visualization to a network trained with triplet loss?

### ISSUE THREAD

**Issue by FSetragno** [tensorflow/lucid#141]
I trained a neural network for image embedding using the triplet loss (like Facenet) and an architecture based on Inception-Resnet. Do the same principles of feature visualization that apply for classification also apply to this kind of task? Can I expect to see some interpretable visualizations?

**Comment by ludwigschubert** [comment]
Sorry, I'm not familiar with this type of loss.

---

Please use the [Stack Overflow tags `tensorflow` & `lucid`](https://stackoverflow.com/questions/tagged/tensorflow+lucid) to ask for help. Stack Overflow provides a better collaborative forum: lots of related questions have been asked there, and some answers may be relevant to you.

When asking for help with a suspected bug, please include a link to demonstrate the issue, preferably as a [Colab notebook](https://colab.research.google.com). It is often impossible to debug from code snippets alone, let alone from error messages or stack traces. Isolate the issue and reduce your code as much as possible before asking for help. Please try to reproduce the bug on one of the models included in `lucid.modelzoo`. The less code you post, the easier it is for someone to debug, and the more likely you are to get a helpful response. 

If you have a question about Lucid’s behavior and want to discuss it with other users, also consider the  `#proj-lucid` channel on the [Distill Community Slack Workspace](https://distillpub.slack.com) which you may [join here](https://join.slack.com/t/distillpub/shared_invite/enQtMzg1NzU3MzEzMTg3LWJkNmQ4Y2JlNjJkNDlhYTU2ZmQxMGFkM2NiMTI2NGVjNzJkOTdjNTFiOGZmNDBjNTEzZGUwM2U0Mzg4NDAyN2E).

Lucid is alpha-status research code, and we appreciate your understanding that this can sometimes mean we will not have the resources to resolve your specific issue—even if it may be an entirely reasonable request in a more mature codebase. Thank you! 🤗
